packages/network/samba/config/smb.conf

```diff
@@ -27,18 +27,13 @@
   security = share
   guest account = root
   socket options = TCP_NODELAY IPTOS_LOWDELAY
-  smb ports = 445
   max protocol = SMB2
   min receivefile size = 16384
   deadtime = 30
-  os level = 20
   mangled names = no
   syslog only = yes
   syslog = 2
-  name resolve order = lmhosts wins bcast host
-  preferred master = auto
```

Comment:
I'd leave the three `master` settings and `os level` in the config even if they're at default values. If people need to create a custom config it's one of the more frequent things to edit.